code-coverage-report/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+plugins {
+  id("com.ibm.wala.gradle.eclipse-maven-central")
```

Comment:
We seem to need to pull in all the Eclipse dependencies or otherwise this doesn't work with Eclipse-related projects